cms/static/js/views/utils/create_library_utils.js

```diff
@@ -1,58 +1,22 @@
 /**
  * Provides utilities for validating libraries during creation.
  */
-define(["jquery", "underscore", "gettext", "js/views/utils/view_utils"],
-    function ($, _, gettext, ViewUtils) {
+define(["jquery", "underscore", "gettext", "js/views/utils/view_utils", "js/views/utils/create_utils_base"],
+    function ($, _, gettext, ViewUtils, CreateUtilsFactory) {
         "use strict";
-        return function (selectors, classes) {
-            var validateTotalKeyLength, setNewLibraryFieldInErr, hasInvalidRequiredFields,
-                createLibrary, validateFilledFields, configureHandlers;
+        var CreateLibrary = function (selectors, classes) {
+            CreateUtilsFactory.apply(this);
```

Comment:
@cahrens Looks like it's an alternative method of setting the prototype correctly. The line that sets the prototype really ought to have been a `new` call. Now that I've seen this, I'm not sure of the exact detailed difference between apply and setting the prototype, but for our purposes, it doesn't matter. I'm going to push a change after rebasing that sticks with calling `new`.